src/Mailer/Message/CitizenInitiativeActivitySubscriptionMessage.php

```diff
@@ -53,7 +54,7 @@ public static function create(
             Uuid::uuid4(),
             '196524',
             $recipient->getEmailAddress(),
-            self::fixMailjetParsing($recipient->getFullName()),
+            MailjetUtils::fixMailjetParsing($recipient->getFullName()),
```

Comment:
the static method still uses `mailjet` keyword.

Reply:
yes i think i will make the ApiClient do this parsing, as they could be different for each api